### PR TITLE
Update TestApp.cpp

### PR DESCRIPTION
**Pull request by toddbellMSFT**
Intellisense was throwing this fix off.

### CODE REVIEW

File: source/test/TestApp/TestApp.cpp [hunk 1]
```diff
@@ -81,7 +81,7 @@ namespace PlayFabUnit
         // Set this up for use when the tests finish
         this->clientApi = std::make_shared<PlayFab::PlayFabClientInstanceAPI>(PlayFab::PlayFabSettings::staticPlayer);
 
-#if !defined(PLAYFAB_PLATFORM_IOS) // IOS proj doesn't include PlatformLoginTest files yet
+#if !defined(PLAYFAB_PLATFORM_GDK) && !defined(PLAYFAB_PLATFORM_IOS) // IOS proj doesn't include PlatformLoginTest files yet
```
Comment (PushpadantK) [inline source/test/TestApp/TestApp.cpp:84]: >IOS [](start = 72, length = 3)

update this comment, is there another bug tracking fix for GDK?

File: source/test/TestApp/TestApp.cpp [hunk 2]
```diff
@@ -92,7 +92,7 @@ namespace PlayFabUnit
         pfApiTest.SetTitleInfo(testTitleData);
         testRunner.Add(pfApiTest);
 
-#if !defined(PLAYFAB_PLATFORM_GDK) && false // These tests are still too unstable, and despite passing nearly 100% in debug, still fail 100% in release
```
Comment (PushpadantK) [inline source/test/TestApp/TestApp.cpp:95]: >false [](start = 38, length = 5)

nit : this condition will always resolve to false anyways.

**Review by PushpadantK — APPROVED** [review]
:shipit:

File: source/test/TestApp/TestApp.cpp (hunk 1 above)
Comment (toddbellMSFT) [reply source/test/TestApp/TestApp.cpp:84]: We will not be pursuing a targeted fix for GDK. I will update the comment.

File: source/test/TestApp/TestApp.cpp (hunk 2 above)
Comment (toddbellMSFT) [reply source/test/TestApp/TestApp.cpp:95]: Yeah, Intellisense was saying it was still being compiled, after I made this fix things ran just fine.

**Pull request merged**